src/events/Windows.cpp

```diff
@@ -412,8 +412,14 @@ void Events::listener_mapWindow(void* owner, void* data) {
 
                     PWINDOW->setHidden(false);
                 } catch (...) { Debug::log(LOG, "Rule move failed, rule: %s -> %s", r.szRule.c_str(), r.szValue.c_str()); }
-            } else if (r.szRule == "center") {
-                PWINDOW->m_vRealPosition = PMONITOR->vecPosition + PMONITOR->vecSize / 2.f - PWINDOW->m_vRealSize.goalv() / 2.f;
+            } else if (r.szRule.find("center") == 0) {
+                auto RESERVEDOFFSET = Vector2D();
+                try {
+                    if (std::stoi(r.szRule.substr(7)) == 1)
```

Comment:
ditto

I also honestly don't like the magic substr here. Can't it just be a CVarList?

Reply:
done